src/azure-cli-core/azure/cli/core/test_utils/vcr_test_base.py

```diff
@@ -355,6 +371,8 @@ def _execute_live_or_recording(self):
             if callable(tear_down) and not self.skip_teardown:
                 self.tear_down()
 
+    @mock.patch('azure.cli.command_modules.datalake.analytics.custom._get_uuid_str',
+                _mock_get_uuid_str)
```

Comment:
Yes this would be an issue. Should be able to add the mock to the tests for datalake where this mock is needed.